src/libCli/MessageParserJson.cpp

```diff
@@ -56,7 +56,7 @@ std::vector<std::unique_ptr<google::protobuf::Message>> MessageParserJson::parse
             buffer << source->rdbuf();
             std::string fileContent = buffer.str();
             auto status = google::protobuf::util::JsonStringToMessage(
-                    google::protobuf::StringPiece(fileContent),
+                    fileContent,
```

Comment:
Actually you don't need a temporay (`fileContent`) here, you could pass it directly as parameter to `JsonStringToMessage()`